jobsdb/jobsdb.go

```diff
@@ -3119,14 +3122,159 @@ func (jd *HandleT) getBackUpQuery(backupDSRange *dataSetRangeT, isJobStatusTable
 	var stmt string
 	if jd.BackupSettings.FailedOnly {
```

Comment:
It is weird that, depending on this flag, the json that we export has a different set of fields...
@chandumlg @cisse21 @lvrach do we use these exports as input anywhere else?

Reply:
I dont think we are exporting these fields anywhere else but for the other question where you say the JSON we export has different fields, Do you mean we store a different JSON schema in S3 based on the flag?

Reply:
yes

Reply:
I can't think of any reason why they should be different. We don't use these table backups even for replay, The backups which we use are `proc_error` which is not a part of this flow and `gw` which doesn't have any Failed jobs. So I feel we can use the same schema

Reply:
What are we using these backups for?

Reply:
I guess most likely they are used for debugging purposes where we need the status table and jobs table to understand what happened. Can't think of any other use case

Reply:
let's leave them as they are then, @saurav-malani I believe you can even skip the trick we are doing for duplicate columns